src/util-landlock.c

```diff
@@ -43,22 +43,22 @@ void LandlockSandboxing(SCInstance *suri)
 static inline int landlock_create_ruleset(
         const struct landlock_ruleset_attr *const attr, const size_t size, const __u32 flags)
 {
-    return syscall(__NR_landlock_create_ruleset, attr, size, flags);
+    return (int)syscall(__NR_landlock_create_ruleset, attr, size, flags);
```

Comment:
syscall returns long, which is 64 bits on 64 bit archs. However this int usage seems to match example code for landlock, and casting syscall returns to other types seems quite normal, in fact the man page example does it as well
```c
           pid_t tid;

           tid = syscall(SYS_gettid);
           syscall(SYS_tgkill, getpid(), tid, SIGHUP);
```
So I guess it is ok?
Might be good to do a `long` type and a bounds check before casting to int though

Reply:
ok

Reply:
with debug validation